### PR TITLE
Section helper story update

### PR DESCRIPTION
**Pull request by justinara**
Story update includes:
- Added component example to page header
- Updated description
- Updated current examples
- Added new examples to showcase component features better

### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6078

File: src/SectionHelper/docs/index.story.js [hunk 1]
```diff
@@ -67,24 +67,97 @@ export default {
 
   sections: [
     header({
-      component: <div style={{ width: '50%' }}></div>,
+      component: (
+        <SectionHelper appearance="standard" fullWidth>
+          Connect to Google with the SEO Wiz and your sitemap index will
+          automatically be submitted.
+        </SectionHelper>
+      ),
```
Comment (EgoziE) [inline src/SectionHelper/docs/index.story.js:75]: We prefer to remove components preview from the header, it's an old thing we did in the past.
Header components cannot be debugged (no code sample to show)
And can be too big, making the header messy

**Comment by milkyfruit** [comment]
@justinara Few notes to discuss:

- Complexity of code examples
- Order of Structure, Appearance... vs building examples from easy to hard
- Layouting: max description width, dividers and titles for groups like appearance, constraints, advanced examples...

File: src/SectionHelper/docs/index.story.js (hunk 1 above)
Comment (argshook) [reply src/SectionHelper/docs/index.story.js:75]: huh that's news to me, didn't know the header is something to go away.

If that's the case, it should be removed from the generator

Comment (EgoziE) [reply src/SectionHelper/docs/index.story.js:75]: Already did long ago

**Pull request merged**